tests/Odbo/AddTest.php

```diff
@@ -73,33 +73,6 @@ public function testAddNullAttribute()
 		$this->assertSame('0', $model->OMU);
 	}
 
-	public function testInsertWithOnlyPrimaryKey()
```

Comment:
The model definition changed, and now has a required column. So this would always fail. And is kind of a redundant test. inserting anything with columns will run the same code as inserting something without columns.